LoopKitUI/ChartColorPalette.swift

```diff
@@ -10,6 +10,8 @@ import UIKit
 
 /// A palette of colors for displaying charts
 public struct ChartColorPalette {
+    public static var defaultPalette = ChartColorPalette(axisLine: .clear, axisLabel: .secondaryLabel, grid: .systemGray3, glucoseTint: .systemTeal, doseTint: .systemOrange)
```

Comment:
Another public static var. This should be a let, probably, if it needs to exist at all.